src/dash/issues/DashIssueDetail.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { useIssue, useIssueArticles } from "../../api/hooks";
+import { Spinner, Table } from "react-bootstrap";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { ArticleTitle } from "../articles/DashArticlesPage";
+
+const DashIssueDetail = () => {
+  const { issueId } = useParams();
+
+  const id = parseInt(issueId || "");
+
+  const [issue] = useIssue(id);
+  const [resp] = useIssueArticles(id);
```

Comment:
At some point we're going need to handle potential errors returned by these hooks.

Reply:
Think we should refactor the get hooks to return the status code somehow. The primary error here is going to be some variation on a 404.

Reply:
We can put status code in APIError.